### PR TITLE
modern-cv:0.1.0

### PR DESCRIPTION
**Pull request by ptsouchlos**
I am submitting
- [x] a new package
- [ ] an update for a package

`modern-cv` is a port of a populate latex CV (resume) template that I was previously using for my own personal resume. I have since moved to typst and wanted to share my ported template. 

Description: This package makes it easy to create a beautiful, modern resume and cover letter. 

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE
- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/modern-cv/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Modern CV
+
+A port of the [Awesome-CV](https://github.com/posquit0/Awesome-CV) Latex resume template in [typst](https://github.com/typst/typst).
+
+## Requirements
+
+You will need the `Robot` and `Source Sans Pro` fonts installed on your system or available somewhere. You can download them from the following links:
```
Comment (reknih) [inline packages/preview/modern-cv/0.1.0/README.md:7]:
```suggestion
You will need the `Roboto` and `Source Sans Pro` fonts installed on your system or available somewhere. You can download them from the following links:
```

Comment (reknih) [reply packages/preview/modern-cv/0.1.0/README.md:7]: If you want you can also add a hint that the web app has both fonts and that there, no further action is necessary.

File: packages/preview/modern-cv/0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        "Software Engineer",
+        "Software Architect"
+      )
+  ),
+  date: datetime.today().display()
+)
+
+= Education
+
+#resume_entry(
```
Comment (reknih) [inline packages/preview/modern-cv/0.1.0/README.md:42]: Normally, Typst uses `kebab-case`

File: packages/preview/modern-cv/0.1.0/template/coverletter.typ [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      phone: "(+1) 111-111-1111",
+      github: "DeveloperPaul123",
+      linkedin: "John Smith",
+      address: "111 Example St. Apt. 111, Example City, EX 11111",
+      positions: (
+        "Software Engineer",
+        "Full Stack Developer"
+      )
+  ),
+  profile_picture: image("./profile.png")
```
Comment (reknih) [inline packages/preview/modern-cv/0.1.0/template/coverletter.typ:17]: Same here and below

File: packages/preview/modern-cv/0.1.0/typst.toml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+[package]
+name = "modern-cv"
+version = "0.1.0"
+compiler = "0.11.0"
+entrypoint = "lib.typ"
+authors = ["Paul Tsouchlos <https://github.com/DeveloperPaul123>"]
+license = "MIT"
+description = "A modern resume template based on the Awesome-CV Latex template."
+keywords = ["CV", "Resume", "Minimalistic", "Professional", "Modern"]
```
Comment (reknih) [inline packages/preview/modern-cv/0.1.0/typst.toml:9]: Add "Curriculum Vitae" for discoverability?

File: packages/preview/modern-cv/0.1.0/lib.typ [hunk 5]
```diff
@@ -0,0 +1,546 @@
...
+    author: author.firstname + " " + author.lastname, 
+    title: "resume",
+  )
+  
+  set text(
+    font: ("Source Sans Pro"),
+    lang: "en",
+    size: 11pt,
+    fill: color_darkgray,
+    fallback: false
```
Comment (reknih) [inline packages/preview/modern-cv/0.1.0/lib.typ:352]: Why did you choose to disable font fallback? It helps users that need wide charsets for their CV, e.g. in Asia.

**Comment by DeveloperPaul123** [comment]
@reknih Thanks for the thorough review! I think I've addressed all the items you brought up.

**Comment by reknih** [comment]
Thank you!

**Pull request merged**